group_vars/all.yml

```diff
@@ -181,11 +181,11 @@ use_custom_keystores: false
 # to default ansible merging behaviour
 dependencies_version:
   postgresql_connector: 42.6.1
-  postgres_major_version: 15
```

Comment:
the last release we did included 15, if now we are reverting this is a breaking change for existing environments

I am not sure how but if it's required only for centos 7 then only centos 7 should stick to 14

Reply:
the only less ugly thing I can think of, is to add an assert task in the pre-checks playbook that if using postgres and current os is centos 7 and current postgres > 14 then they should downgrade

Reply:
I'm not sure what use case you are trying to cover for. If users specify a postgres host then it should not have any postgres installation on it - neither from a manual installation, nor from a a run of a previous version of the playbook - as that's not something we support